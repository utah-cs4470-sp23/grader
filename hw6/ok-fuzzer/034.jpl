show (if (true != ((if true then 19.28 else 97.63) <= 2.56)) then [[pict., pict.]] else [[pict.], (if (! false) then [pict., pict., pict.] else [pict., pict., pict.]), [pict., pict.]])[((738 / (- 851)) % (- 820))]

