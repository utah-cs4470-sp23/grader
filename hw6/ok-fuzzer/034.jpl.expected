(ShowCmd
 (ArrayIndexExpr
  (ArrayType
   (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
   1)
  (IfExpr
   (ArrayType
    (ArrayType
     (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
     1)
    1)
   (BinopExpr
    (BoolType)
    (TrueExpr)
    !=
    (BinopExpr
     (BoolType)
     (IfExpr
      (FloatType)
      (TrueExpr)
      (FloatExpr (FloatType) 19)
      (FloatExpr (FloatType) 97))
     <=
     (FloatExpr (FloatType) 2)))
   (ArrayLiteralExpr
    (ArrayType
     (ArrayType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
      1)
     1)
    (ArrayLiteralExpr
     (ArrayType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
      1)
     (VarExpr
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
      pict.)
     (VarExpr
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
      pict.)))
   (ArrayLiteralExpr
    (ArrayType
     (ArrayType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
      1)
     1)
    (ArrayLiteralExpr
     (ArrayType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
      1)
     (VarExpr
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
      pict.))
    (IfExpr
     (ArrayType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
      1)
     (UnopExpr (BoolType) ! (FalseExpr))
     (ArrayLiteralExpr
      (ArrayType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       1)
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       pict.)
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       pict.)
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       pict.))
     (ArrayLiteralExpr
      (ArrayType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       1)
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       pict.)
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       pict.)
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       pict.)))
    (ArrayLiteralExpr
     (ArrayType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
      1)
     (VarExpr
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
      pict.)
     (VarExpr
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
      pict.))))
  (BinopExpr
   (IntType)
   (BinopExpr
    (IntType)
    (IntExpr (IntType) 738)
    /
    (UnopExpr (IntType) - (IntExpr (IntType) 851)))
   %
   (UnopExpr (IntType) - (IntExpr (IntType) 820)))))
